src/components/NextTrack.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-subheader v-if="hasNext" class="v-btn overline">
+    <span class="font-weight-regular">Next:</span>
+    <span class="custom-transform-class text-none font-weight-bold">{{trackData.title}}</span>
+    <span class="custom-transform-class text-none font-weight-thin"> von </span>
+    <span class="custom-transform-class text-none">{{trackData.artist}}</span>
+  </v-subheader>
+</template>
+
+<script>
+import axios from "axios";
+import { mapGetters } from "vuex";
+
+const POLL_TIMEOUT = 3000;
+
+export default {
+  name: "NextTrack",
+  created() {
+    let poll = async () => {
+      try {
+        let response = await axios.get("/data/prio.m3u", {
```

Comment:
`prio.m3u` is gone. Use `"/api/player/queue/"` instead.

Reply:
I'm also gonna test if it works as intended, last I checked UX wasn't consistent with actual behavior,

Reply:
(how deployed are the backend changes? 🥺)

Reply:
It is deployed like: it is rönning in prodökschn

Reply:
You get the following when `GET`-ing `/api/player/`: 
```json
{
  "uptime": "0d 19h 04m 39s",
  "liquidsoap_version": "Liquidsoap 1.3.7",
  "api_version": "0.0.15",
  "music": "c884fc50-b600-425d-bda3-b8b655fd2f03",
  "classics": "d0798bcd-e345-4742-9171-8ac78262f666",
  "jingles": "8d5ea818-2667-41b9-9b1b-42e071540bf1",
  "on_air": true,
  "current_track": {
    "id": "81922c26-43a3-4040-b60a-363269eacd02",
    "remaining": 297.4,
    "source": "music"
  },
  "queue": ""
}
```

For every playlist there is the id of the next scheduled track, plus the next scheduled track in the queue, if any.

Reply:
thanks for the pointer, i gonna test if the now playing feature follows the behaviour i would expect while i update this branch